backend/models/schemas.py

```
@@ -0,0 +1,97 @@
+            datetime.strptime(v, '%Y-%m-%d')
+            return v
+        except ValueError:
+            raise ValueError('Birth date must be in YYYY-MM-DD format')
+    
+    @validator('birth_time')
+    def validate_birth_time(cls, v):
+        try:
+            datetime.strptime(v, '%H:%M')
+            return v
+        except ValueError:
+            raise ValueError('Birth time must be in HH:MM format')
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)